docs/deployment/getting-started.md

```diff
@@ -0,0 +1,76 @@
+# Deployment
+
+Server deployment is a complex area, that will depend on what kind of service you're deploying Uvicorn onto.
+
+### TL;DR
+As a general rule, you probably want to:
+
+* Run `uvicorn --reload` from the command line for local development.
+* Run `gunicorn -k uvicorn.workers.UvicornWorker` for production (non containerized).
```

Comment:
why non-containerized ? some people may like to have it thar way and it's a viable option,